site/documentation/content/user-guide/mqtt-adapter.md

```diff
@@ -274,25 +274,38 @@ The following variables are used:
 
 ### Receiving Commands (authenticated Device)
 
-An authenticated device MUST use the topic filter `command///req/#` to subscribe to commands.
+An authenticated device MUST use one of the following topic filters for subscribing to commands:
+
+* `command///req/#`
+* `command/${tenant-id}//req/#`
+* `command/${tenant-id}/${device-id}/req/#`
```

Comment:
Strictly speaking, usage of `command//${device-id}/req/#` would work as well. So, maybe better to just use
`command///req/#`
`command/${tenant-id}/${device-id}/req/#`
noting that in the latter case  either tenant or device id may either be set to the value of the authenticated device or may be kept empty.